packages/chart/src/components/LineChart/components/LineChart.BumpCircle.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react'
+import { Group } from '@visx/group'
+import { type Column } from '@cdc/core/types/Column'
+import React from 'react'
+
+
+const LineChartBumpCircle = props => {
+  const { config, xScale, yScale, parseDate } = props
+  const [tooltipID] = useState<string>('')
+
+  const handleX = xValue => {
+    if (config.xAxis.type === 'date') {
+      return parseDate(xValue).getTime()
+    }
+    if (config.xAxis.type === 'date-time') {
+      return new Date(xValue)
+    }
+    if (config.xAxis.type === 'categorical') {
+      return xValue
+    }
+  }
+
+  const checkBandScale = xValue => {
+    return xScale.bandwidth ? xScale.bandwidth() / 2 + Number(xValue) : Number(xValue)
+  }
+
+  const getListItems = dataRow => {
+    const listItems = []
+
+    Object.values(config.columns)?.forEach((column: Column) => {
+      if (!column.tooltips) return
+      if (!column.label) {
+        listItems.push(
+          `<li className='tooltip-body'>
+            <strong>${column?.name}</strong>: ${dataRow[column.name]}
+          </li>`
+        )
+      } else {
+        listItems.push(
+          `<li className='tooltip-body'>
+            <strong>${column?.label}</strong>: ${dataRow[column.name]}
+          </li>`
+        )
+      }
+    })
+    return listItems.join(' ')
+  }
+
+  const getTooltip = dataRow => `<ul>
+                  <li class="tooltip-heading">${tooltipID}</li>
```

Comment:
This is never updated so it should be removed.

Reply:
removed  **<li class="tooltip-heading">${tooltipID}</li>** line